homework-submission/week4/src/components/TODOList.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable */
+import React, { Component } from 'react';
+
+class TODOList extends Component {
+  render() {
+    const TODOList = this.props.TODOListItems.map((element, index) =>
+      <div key={index}>

```

Comment:
index is a good idea to use here but if you can use something else instead it would be better in avoiding Bugs cases